### PR TITLE
New design for the error page

### PR DESCRIPTION
**Pull request by meles5**
Just a simple design update for the error page.

### CODE REVIEW

File: qutebrowser/html/error.html [hunk 1]
```diff
@@ -35,19 +69,29 @@
...
...vpYK8Y1AvmguK+DKzziztAtES26UsdgjkZpfPnkaAlJETMgjkzW1LKtahZ4QnXM6TkbQB39v3lcMFk7qECYUYY9Qa4jQSXP7/Jd0Ws6oDXRy+C8WZIOjJSNdIJhgAmAnwpmQcTgB6r0tZQtx6ST0tAwlLGfNGWQSmE0oJeML9UiDm4SYIWV7dqP37gOnIAfNeortL5XxOEJ0QwGbkiAuiok8zvhRUwpP0RgEN9fK/IUepwTzemGwW4/LcbzetiUQea/if5CDNWMtDKodwlhRHgiUkwi4lYBCMBeQlAj74sn9bNkK9J5tOSmEvTB47hoi3ZbkL6JTSFGHyEJH3v/qnazxdf5yBn98j5hmvGxG8R0SIiyon5DyRUmd72j8L6jFa1HoyduISFQNeDZPlvqno+FYvvN3RQyp/H54jVU/OIC1MJSarDjVUU2TsBY7KyunOQkASgx4KY0nwTzBWn6b0tIX2+mGXpNvMlZGVHtWQsvG+a/iwRXXxAxj2SYmThcWJaQMfzDMQ0O2cJKddi2+/CWxln79NeZmzujQ4ImOSFg/AREUhQRPQJRFeCRPnzGzy3xpourL+Leu4Zg5V3jlb3PjBRxf0TVHx8qEBBSjgUL0SDSRho2HZRAvISgB4T0vnD0Q3S5tdY8mnMz3+WHps5CZp6GF2XQNAlM3aAeMGDxcbzjm4Y91CWO1UsEgLfJkHuOPnMRoJ4NxIPksRbgNDlOo9XR0wCTSNFPPGbjd45sfZuXyx2v6cq9IJbhRySLvCxoQo+P1bFtYUC6XoI2Xqo2DsjlVVlXALyEoAeS8DxVwa2nPa3GD5Lb/YARzv7t1+7ZFkFQQvuv9xY5eiG4rkFrtSkxSDxtchWFguS4jDt6kFkTOFk27sBGXPZ0yzLgtfrizCoYxKIyn9d5b0npjYbRMwe+3eQWAv2503PSxK4oUjB7DEqJg8gqBRGYO8ze2eVwJPxCitISALQY0HafjS0GpB/Znm6OdI2TbBlx5wd2mcB1a0SkvsnqDPzuwRe8MBU7c9Ofm+UlBW5VKOcCI/2NSwrJNAQ1O2c7QJPjsSbelXXUQbtirXx9ni8aGuPfE5wIoxRQOW/2eR9KKZY+gz3fgiswCm+PQKE0ZkCt49S8ZkRCgpTKexOb07YO/kD/selH0ZqAvYSgB47IGLTXwB8eDpNlLBj9Cx9Xwuj3dsvz8zftK
```
Comment (meles5) [inline qutebrowser/html/error.html]: base64-encoded image

**Comment by The-Compiler** [comment]
Here is how the error page looks with this PR:

errpage

I really like the design, but I don't like the text :wink:

I don't think the list is necessary - this is helpful for browsers with non-technical users, but for qutebrowser this will just lead to people not actually looking at the error message.

That in turn is too cramped with the other red error text. What about something simpler like this instead?

errpage

As for the image, instead of base64-encoding it you can do something this instead:

* Use `{{ logo_url }}` in the template
* Do this in `browser/webpage.py` in `_handle_errorpage`:

```python
logo = utils.resource_filename('img/broken_qute_logo.png')
logo_url = QUrl.fromLocalFile(logo).toString(QUrl.FullyEncoded)
html = template.render(..., logo_url=logo_url)
```

See how it's done in `qutebrowser/html/dirbrowser.html` and `qutebrowser/browser/network/filescheme.py`.

**Comment by meles5** [comment]
Thank you for the help, I had no time over the week, but I'll try to change the error page over the weekend.

I could even try to create a .svg file out of this current .xcf image - because an image with a fixed size is not the best solution.

**Comment by The-Compiler** [comment]
No worries - I'll probably have no time on the weekend though :laughing: An `.svg` would be great, yeah!

**Comment by meles5** [comment]
No promises if it works - I'll just try Inkscape or other SVG Editors to render an .png as an .svg file, hope that this will work

**Comment by The-Compiler** [comment]
That won't work, no - you can't just vectorize a rastered image (not in a way that it's usable, anyways). You'd have to take the SVG logo and re-create your edits in Inkscape instead of Gimp.

**Comment by meles5** [comment]
I could try it - but [this one](https://github.com/LFUnion/left/blob/master/icons/left-logo.svg) is made off an .png file

**Comment by The-Compiler** [comment]
I can imagine it vectorization works with logos like this, where you have clear, uniform, simple colored areas. I don't think it will work with the qutebrowser logo.

But as said it's SVG to begin with - so if you know Inkscape more than me (or want to learn it), I think editing the original SVG in Inkscape is the way to go. If not, I guess the .png will work too.

**Comment by meles5** [comment]
Should work now - And sorry for the old-school .png file - maybe I am able to understand how to use Inkscape in a few days...

**Comment by The-Compiler** [comment]
It seems that broke a test which checks for the `errorTitleText` id - can you adjust the test accordingly? (If you add an id, it should probably be called `error-title-text` - but you could also adjust the test so it doesn't need an id)

**Pull request closed without merging**